reference/data/models/bolus/deliverycontext.v1.yaml

```diff
@@ -0,0 +1,8 @@
+title: Bolus Delivery Context
```

Comment:
Nit: this could benefit from an additional long-form `description` to explain the enums.